map/bookmark_manager.cpp

```diff
@@ -1858,15 +1858,15 @@ void BookmarkManager::LoadBookmarkRoutine(std::string const & filePath, bool isT
       if (m_needTeardown)
         return;
 
-      base::DeleteFileX(fileSavePath);
+      base::DeleteFileX(pathToLoad);
       if (kmlData)
       {
-        fileSavePath = GenerateValidAndUniqueFilePathForKML(base::GetNameFromFullPathWithoutExt(std::move(fileSavePath)));
+        pathToLoad = GenerateValidAndUniqueFilePathForKML(base::GetNameFromFullPathWithoutExt(std::move(pathToLoad)));
```

Comment:
It's better to create a new variable here instead of reusing the previous one.

Reply:
Fixed